lib/Mojolicious/Command/prefork.pm

```diff
@@ -23,12 +23,21 @@ sub run {
     'k|keep-alive-timeout=i' => sub { $prefork->keep_alive_timeout($_[1]) },
     'l|listen=s'             => \my @listen,
     'P|pid-file=s'           => sub { $prefork->pid_file($_[1]) },
-    'p|proxy'                => sub { $prefork->reverse_proxy(1) },
+    'p|proxy:s'              => \my @proxy,
     'r|requests=i'           => sub { $prefork->max_requests($_[1]) },
     's|spare=i'              => sub { $prefork->spare($_[1]) },
     'w|workers=i'            => sub { $prefork->workers($_[1]) };
 
   $prefork->listen(\@listen) if @listen;
+  $prefork->reverse_proxy(1) if @proxy;
+  my @trusted = grep {length} @proxy;
+  $prefork->trusted_proxies(\@trusted) if @trusted;
```

Comment:
Why not just `$prefork->trusted_proxies([grep {length} @proxy]);`?

Reply:
I guess I was saving a method call if it wasn't needed, but this isn't going to be a hot code path, so it doesn't matter to me

Reply:
But I guess that first one handles all of this so, ok